pkg/apis/serving/v1alpha1/kfservice_framework.go

```diff
@@ -52,6 +52,17 @@ func (m *ModelSpec) Validate() error {
 	return err
 }
 
+type FrameworkConfig struct {
+	ContainerImage string `json:"image"`
+
+	//TODO add readiness/liveness probe config
```

Comment:
I'm a little wary of opening the floodgates to readiness/liveness configurability. That's one of our value props.